databricks/koalas/tests/test_dataframe.py

```diff
@@ -282,6 +282,11 @@ def test_dropna(self):
         with self.assertRaisesRegex(NotImplementedError, msg):
             ddf.dropna(axis='foo')
 
+    def test_dtype(self):
+        d = self.df
+        full = self.full
```

Comment:
actually, you are only testing a single type here. Can you make a small dataframe with a bunch of different types in it? The pandas test suites have a few example. No need to support all the complex types, at least the basic scalar types.